x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_features/stream_features/stream_existing_features_table.tsx

```diff
@@ -87,7 +87,18 @@ export function StreamExistingFeaturesTable({
       width: '15%',
       render: (feature: Feature) => {
         if (isFeatureWithFilter(feature)) {
-          return <FeatureEventsSparkline feature={feature} definition={definition} />;
+          const now = Date.now();
+          return (
+            <FeatureEventsSparkline
+              feature={feature}
+              definition={definition}
+              timeRange={{
```

Comment:
Should we have this timeRange shared between the 2 components? I'm assuming that if one changes the other should change as well.

Also I'd probably use [moment](https://www.npmjs.com/package/moment) to avoid magic numbers and make the code more readable.